### PR TITLE
Implement `quilt config`

### PR DESCRIPTION
**Pull request by dimaryaz**
Sets the registry URL. The config can still be overridden using the env var.

### CODE REVIEW

**Comment by dimaryaz** [comment]
Ping?

File: quilt/tools/command.py [hunk 1]
```diff
@@ -51,25 +52,80 @@
...
+
+def get_auth_path():
+    url = get_registry_url()
+    if url == DEFAULT_REGISTRY_URL:
+        suffix = ''
+    else:
+        # Store different servers' auth in different files.
+        suffix = "-%.8s" % hashlib.md5(url.encode('utf-8')).hexdigest()
+
+    return os.path.join(BASE_DIR, 'auth%s.json' % suffix)
```
Comment (kevinemoore) [inline quilt/tools/command.py]: Shouldn't the config and the login/auth be stored in the same folder? Both seem like user_config rather than user_data.

Comment (dimaryaz) [reply quilt/tools/command.py]: Well, it's kinda subjective. I was thinking of auth as "state" (that the user should never touch directly), while the registry URL (and soon analytics switch) is "config" that the user gets to edit.

That said, two different directories might be overkill. If we use one, it's probably better to keep using `BASE_DIR` - it's `user_data_dir`, so somewhat neutral, I guess.

Comment (kevinemoore) [reply quilt/tools/command.py]: Let's keep it simple and use a single directory. That'll make it easy for someone, including us, to wipe out our config and auth data and start fresh.

**Pull request merged**